package.json

```diff
@@ -107,6 +121,13 @@
           "group": "fluttergpt-menu-group@1"
         }
       ],
+      "explorer/context": [
+        {
```

Comment:
@tanmoy27112000 what's the use case of having the command here? I tried removing it and didn't see any difference. anything I am missing?